mandatory/1-fix-functions.js

```diff
@@ -22,31 +21,27 @@ function mood() {
 
 function greaterThan10(num) {
   let isBigEnough;
-
-  if (isBigEnough) {
+  if (isBigEnough > 10) {
     return "num is greater than 10";
   } else {
     return "num is not big enough";
   }
 }
 
 function sortArray(letters) {
-  let sortedLetters = letters;
-
+  let sortedLetters = letters.sort();
   return sortedLetters;
 }
 
 function first5(numbers) {
```

Comment:
let is assigned undefined first and only after one line it is reassigned. What do you think if we rewrite line 38 as `let sliced = numbers.slice(0, 5);` then delete the line 37?

Reply:
Very true, Done. Thank you :)